src/main/java/application/App.java

```diff
@@ -65,44 +66,24 @@ private static Router createRouter(String directory) {
 
         app.post("/dog", (Request request, Response response) -> {
             String uniqueRoute = app.getUniqueRoute(request.getRoute());
-            String resourceRoute = app.saveResource(uniqueRoute, request.getContentFileType(),
+            OperationResult saveResult = app.saveResource(uniqueRoute, request.getContentFileType(),
                     request.getBody());
-            response.forPost(resourceRoute);
+            response.forPost(saveResult, uniqueRoute);
         });
 
-        app.put("/cat/1", (Request request, Response response) -> {
-            app.saveResource(request.getRoute(), request.getContentFileType(), request.getBody());
-            response.forPut();
+        app.put("/cat/:id", (Request request, Response response) -> {
+            OperationResult result = app.saveResource(request.getRoute(), request.getContentFileType(),
+                    request.getBody());
+            response.forPut(result);
         });
 
-        app.get("/multiple_parameters", (Request request, Response response) -> {
-            String body = "Parameters: \n" + request.getParameters().entrySet();
-            response.setBody(body, MIMETypes.plain);
+        app.get("/cat/:id", (Request request, Response response) -> {
```

Comment:
Do you think it makes more sense to order the routes starting with GET? Like it's put then get—to me, get first makes more sense to what I've seen unless you have diff reasoning for it